client/src/store.js

```diff
@@ -4,9 +4,9 @@ const storedVotedFor = JSON.parse(localStorage.getItem("votedFor"));
 export const votedFor = writable(!storedVotedFor ? {} : storedVotedFor);
 votedFor.subscribe(value => {
     if (value) {
-        localStorage.setItem("votedFor", JSON.stringify(value));
+	localStorage.setItem("votedFor", JSON.stringify(value));
```

Comment:
I'm assuming making this a tab wasn't intentional?

Reply:
Not intentional, I think I opened the project in code and it must have done the conversion automatically. I'll revert those  once I get home.